public/components/trace_analytics/components/services/service_view.tsx

```diff
@@ -106,25 +103,19 @@ export function ServiceView(props: ServiceViewProps) {
     );
 
     setIsServiceOverviewLoading(true);
+    setIsServicesDataLoading(true);
     handleServiceViewRequest(
       props.serviceName,
       props.http,
       DSL,
       setFields,
       mode,
+      setServiceMap,
       props.dataSourceMDSId[0].id
-    ).finally(() => setIsServiceOverviewLoading(false));
-
-    if (mode === 'data_prepper' || mode === 'custom_data_prepper') {
```

Comment:
I'm understanding that the content of this condition has the service map call you're removing, but is the actual condition here (i.e. `mode === <something>`) being removed problematic to leave out? handleServiceViewRequest doesn't have that same condition above.

Reply:
This was previously just preventing the call if the mode was set to jaeger, as the service map is not displayed in this mode. As the existing logic already calls handleServiceMapRequest(). 
The previous logic was actually still calling this function just once instead of twice for the jaeger mode.
The condition is not necessary in the new code as the setServiceMap is only utilized in displaying the service map which is not present in jaeger.